initfiles/bash/etc/init.d/install-init.in

```diff
@@ -300,3 +300,13 @@ fi
 if [ -d ${CONFIG_DIR} ]; then
     date > ${CONFIG_DIR}/installed
 fi
+
+if [ ! -d ${LOG_DIR} ]; then
+   mkdir -p ${LOG_DIR}
+   chown ${user}:${user} ${LOG_DIR}
+fi
+[ -e  ${LOG_DIR}/hpcc-init.debug ] && chown ${user}:${user} ${LOG_DIR}/hpcc-init.debug
+
+createRuntime
```

Comment:
Actually "$compName", referenced by createRuntime, is not defined so install may throw error unless modifying createRuntime function. As user may change HPCC cluster topology I wonder if it is worthy we process the runtime setting during install.